_posts/2016-07-09-tomesh-mini.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Let's get rid of this post for now. The game play and backstory is still being discussed. We'll publish a revised version next weekend.